lighthouse-core/scripts/legacy-javascript/run.js

```diff
@@ -191,7 +76,8 @@ async function createVariant(options) {
     fs.writeFileSync(`${dir}/.babelrc`, JSON.stringify(babelrc || {}, null, 2));
     // Not used in this script, but useful for running Lighthouse manually.
     // Just need to start a web server first.
-    fs.writeFileSync(`${dir}/index.html`, `<title>${name}</title><script src=main.bundle.min.js>`);
```

Comment:
lack of closing script element = bad sad times w/ no debuggability in CDT. chrome still requests the script, still shows in Network as a script, yet never loads.